aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -0,0 +1,206 @@
+// Copyright © Aptos Foundation
```

Comment:
What's happening with this file? I see the gas-related changes above, but this seems like a new file (and somewhat maybe auto-generated)? should it be like this or is it some rebase issue?

Reply:
I think this file was removed by Victor in the main branch. And for some reason, the file still existed on this branch. I removed it now.